examples/react-app/src/index.tsx

```diff
@@ -13,20 +12,17 @@ import {
   applyMode,
 } from '@cloudscape-design/global-styles';
 
+import "@cloudscape-design/global-styles/index.css"
+import '@iot-app-kit/components/styles.css';
+
```

Comment:
Same here, why we are importing cloudscape and styles in an application this should bundled and a customer who uses our component doesn't have to import these explicitly in their APP.

Reply:
But this means every customer who wants to use our Knowledge Graph component in their App they have to import cloudscape global styles. This is a very strange limitation and looks very bad. if this is the case then I will move knowledge Graph component from reac-components to its own component like scene composer.